policy/modules/system/libraries.if

```diff
@@ -1,5 +1,24 @@
 ## <summary>Policy for system libraries.</summary>
 
+########################################
+## <summary>
+##	Load firmware files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`libs_load_firmware',`
+	gen_require(`
+		type lib_t;
+		class system firmware_load;
```

Comment:
Also not necessary.

Reply:
It generates a SELint warning otherwise: _(W): No explicit declaration for userspace class system.  You should access it via interface call or use a require block. (W-001)_